src/backend/database_migrations/versions/20220803_224221_add_submitting_lab_to_groups.py

```diff
@@ -0,0 +1,32 @@
+"""add submitting lab to groups
+
+Create Date: 2022-08-03 22:42:27.655366
+
+"""
+import enumtables  # noqa: F401
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "20220803_224221"
+down_revision = "20220621_232148"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "groups",
+        sa.Column("submitting_lab", sa.String(), nullable=True),
+        schema="aspen",
+    )
+
+    conn = op.get_bind()
+    populate_submitting_lab_sql = sa.sql.text(
+        "UPDATE aspen.groups SET submitting_lab = name"
+    )
+    conn.execute(populate_submitting_lab_sql)
+
+
+def downgrade():
+    pass
```

Comment:
can we do this instead of passing? Otherwise it's not clear that the back migration didn't run, also i think it's fairly easy to add the downgrade migration? You just drop the `submitting_lab` column? 
`op.drop_column("groups", "submitting_lab", schema="aspen")`
 
```suggestion
raise NotImplementedError("Reversing this migration is not supported")
```

Reply:
It is easy to add a downgrade, but the consensus we have been working with is to never run downgrades 🤷 @jgadling perhaps you can speak more to this

Reply:
i mean that makes sense.. can we still raise the NotImplementedError though? 😄

Reply:
Yes, I added it